Makefile

```diff
@@ -168,13 +168,13 @@ build-all: $(foreach os,$(OS),build/shipperctl.$(os)-amd64.tar.gz) build/sha256s
 build:
 	mkdir -p build
 
-build/shipper-state-metrics.%-amd64: cmd/shipper-state-metrics/*.go $(PKG)
+build/shipper-state-metrics.%-amd64: $(PKG) cmd/shipper-state-metrics/* 
```

Comment:
nit: you have a trailing whitespace here.

Reply:
This was done by Julio, his explanation was:
Without this, `make build/shipperctl.darwin-amd64` wouldn't rebuild
shipperctl even if files in `cmd/shipperctl/cmd/` changed.